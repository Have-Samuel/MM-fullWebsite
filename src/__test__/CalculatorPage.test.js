import renderer from 'react-test-renderer';
import CalculatorPage from '../pages/CalculatorPage';

it('render correctly', () => {
  const tree = renderer.create(<CalculatorPage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
