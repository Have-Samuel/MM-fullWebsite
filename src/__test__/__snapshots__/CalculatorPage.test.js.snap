// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  className="main-content calculator"
>
  <div>
    <h1>
      Let's do some math!
    </h1>
  </div>
  <div
    className="calculator"
  >
    <div
      className="display"
    >
      0
    </div>
    <div
      className="keyboard"
    >
      <div
        className="keyboard-row"
      >
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          AC
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          +/-
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          %
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          รท
        </button>
      </div>
      <div
        className="keyboard-row"
      >
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          7
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          8
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          9
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          x
        </button>
      </div>
      <div
        className="keyboard-row"
      >
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          4
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          5
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          6
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          -
        </button>
      </div>
      <div
        className="keyboard-row"
      >
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          1
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          2
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          3
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          +
        </button>
      </div>
      <div
        className="keyboard-row"
      >
        <button
          className="keyboardBtn"
          id="zero"
          onClick={[Function]}
          type="button"
        >
          0
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          .
        </button>
        <button
          className="keyboardBtn"
          id=""
          onClick={[Function]}
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>
</div>
`;
